pkg/syncer/differ/namespace_test.go

```diff
@@ -39,9 +39,10 @@ func markForDeletion(nsConfig *v1.NamespaceConfig) *v1.NamespaceConfig {
 }
 
 var (
-	disableManaged    = syncertest.ManagementDisabled
-	managementInvalid = core.Annotation(metadata.ResourceManagementKey, "invalid")
-	managementEmpty   = core.Annotation(metadata.ResourceManagementKey, "")
+	enableManaged     = metadata.WithManagementMode(metadata.ManagementEnabled)
```

Comment:
nit: this might be a useful abstraction in the `metadata` package, i.e. instead of `metadata.WithManagementMode` we could have `metadata.WithManagementEnabled` and `metadata.WithManagementDisabled`

Reply:
I considered this, but there's also a `syncertest.ManagementEnabled` that does something different: adding the resource-id annotation and managed-by label. That seems overloaded for that name, but I wasn't sure what to rename it to. So I didn't want to make two methods with the same name in different packages doing subtly different things.